Course/IOS15(P1)/Lesson3/Styles.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  style.swift
+//  Course
+//
+//  Created by Марк Горкій on 11.10.2022.
+//
+
+import Foundation
+import SwiftUI
+
+struct StrokeStyle: ViewModifier {
```

Comment:
`StrokeStyle` and `StrokeStyleL1` are the same. In general, you should follow DRY (Don't repeat yourself) principle